compiler/rustc_expand/src/config.rs

```diff
@@ -117,6 +117,12 @@ pub fn features(sess: &Session, krate_attrs: &[Attribute], crate_name: Symbol) -
             // Otherwise, the feature is unknown. Record it as a lib feature.
             // It will be checked later.
             features.set_declared_lib_feature(name, mi.span());
+
+            // Similar to above, detect internal lib features to suppress
+            // the ICE message that asks for a report.
+            if features.internal(name) && ![sym::core, sym::alloc, sym::std].contains(&crate_name) {
```

Comment:
Gotta do this after `set_declared_lib_feature` or else `Features::internal` ICEs 😃

Reply:
Why the exceptions for std/alloc/core? Users won't usually be building those anyway...

Reply:
This is also pre-existing. I copied this code from above.

Reply:
If we want to fix the exemptions for these libraries we should do it in another PR.

Reply:
Ah, I didn't realize that this is pre-existing. Odd, but :shrug: